cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/IRVariable.qll

```diff
@@ -158,18 +193,28 @@ class IRTempVariable extends IRGeneratedVariable, IRAutomaticVariable, TIRTempVa
   override string getBaseString() { result = "#temp" }
 }
 
+/**
+ * The temporary variable generated to hold the return value of a function.
+ */
 class IRReturnVariable extends IRTempVariable {
   IRReturnVariable() { tag = ReturnValueTempVar() }
 
   final override string toString() { result = "#return" }
 }
 
+/**
+ * The temporary variable generated to hold the exception thrown by a `ThrowValue` instruction.
+ */
 class IRThrowVariable extends IRTempVariable {
   IRThrowVariable() { tag = ThrowTempVar() }
 
   override string getBaseString() { result = "#throw" }
 }
 
+/**
+ * The variable generated to represent the contents of a string literal. This variable acts much
```

Comment:
Thanks for adding all this QLDoc!

The QLDoc for classes in this file starts with "The" or "Represents a", but our [QLDoc style guide](https://wiki.semmle.com/pages/viewpage.action?spaceKey=IN&title=QLDoc+Style+Guide) recommends starting with "A". I think it would be clear and consistent to follow the style guide here.

Reply:
Fixed. I had used the same convention in other files, but I'll change those in a separate PR.